transformation.go

```diff
@@ -439,6 +439,16 @@ func (it *ImageTransformation) Gamma(gamma float64) error {
 	}
 }
 
+// Change (or enforce) the given interpretation/colorspace.
+func (it *ImageTransformation) ChangeColorspace(interpretation Interpretation) error {
+	if image, err := vipsColourspace(it.image, interpretation); err != nil {
```

Comment:
so this will declare `image` and `err` in the scope of the `if` and `else` blocks only, right?

Reply:
Exactly, yep.